src/Facebook/InstantArticles/Elements/GeoTag.php

```diff
@@ -101,4 +106,14 @@ public function toDOMElement($document = null)
 
         return $element;
     }
+
+    /**
+     * Overrides the @see Element::isValid().
+     *
+     * @return true for valid GeoTag that contains not empty script, false otherwise.
+     */
+    public function isValid()
+    {
+        return !Type::isTextEmpty($this->script);
```

Comment:
This should be validated to not only a valid JSON but also a valid GeoJSON: http://geojson.org/

Opened Issue: #99